core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -218,6 +218,13 @@ public String toString() {
           .setDefaultValue(false)
           .setDescription("Flag used only during tests to allow special behavior.")
           .build();
+  public static final PropertyKey TMP_DIRS =
+      new Builder(Name.TMP_DIRS)
+          .setDefaultValue("/tmp")
+          .setDescription("The path(s) to store Alluxio temporary files, such as files to be "
```

Comment:
Does it make sense to specify that the netty jar is not using this location? (Instead it uses `java.io.tmpdir`). Or rename the property to `alluxio.ufs.tmp.dirs` to indicate its use more precisely

Reply:
I've clarified the description. I prefer to leave it as tmp dirs since I think we can use it to better effect in the future.

Reply:
sgtm